dataland-community-manager/communityManagerOpenApi.json

```diff
@@ -28,6 +28,16 @@
         "summary": "Send a single request",
         "description": "A single of data requests for specific frameworks and companies is being sent.",
         "operationId": "postSingleDataRequest",
+        "parameters": [
```

Comment:
Probleme im funktionalen Review:

- Der impersonated request landet in der DB mit der ID des admins
- Man kann keinen impersonate request stellen wenn der admin einen analogen request bereits hat
- Wenn der Status resolved oder nonsourcable ist, gibt es kein update bei neuen Daten. Warum?